CCache/Makefile.in

```diff
@@ -8,6 +8,7 @@ bindir=@bindir@
 mandir=@mandir@
 INSTALLCMD=@INSTALL@
 PACKAGE_NAME=@PACKAGE_NAME@
+MYNAME=@MYNAME@
```

Comment:
I don't really like `MYNAME`, could you change it to `PROGRAM_NAME` please in configure.ac and Makefile and leave the C source as is? (I don't want the C source to unnecessarily diverge from the original).